eosc/cmd/systemNewaccount.go

```diff
@@ -44,6 +44,8 @@ active:
     weight: 1
 ---
 
+For use with any chain other than the EOS mainnet, add the flag `--contract [contract_name]`
+and be sure to add the correct symbol and precision (e.g. "1.0000 SYS") for the contract.
```

Comment:
If inferred correctly, "1", "1.00" and the likes will all work correctly. So we might want to "loosen" a bit the description.